operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -43,6 +50,7 @@ spec:
 #      - "openshift.*"
 #      - "ibm.*"
 #      - "kiali.*"
```

Comment:
As an aside, as we discussed, perhaps kiali.* is to broad and it should be just "kiali-operator".